src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -203,6 +203,12 @@ def test_create_advanced_sequence(single_channel_session):
     single_channel_session._create_advanced_sequence(sequence_name='my_sequence', attribute_ids=ids, set_as_active_sequence=True)
 
 
+def test_lock_context_manager(single_channel_session):
```

Comment:
What's special about nidcpower that you added only to its system test?
How does it test that the lock gets released?

Should be enough to put this in nifake unit tests, I would think.

Reply:
Originally I was unsure if unit tests would be enough. I am now convinced that they are so removed this test